### ISSUE TITLE
Polymer starter kit 1.1.1 getting an error when using gulp

### ISSUE THREAD

**Issue by johnwhitton** [PolymerElements/polymer-starter-kit#517]
Hi,

I just downloaded the latest polymer starter kit (1.1.1).
I then ran
*$ sudo npm install -g gulp bower && npm install && bower install
$ npm install gulp -g
$ gulp serve*
Which worked fine
However when I run 
*$ gulp*
I get the error below.
Note I was doing this to establish a baseline to help resolve an issue I was having when using firebase similar to https://github.com/Polymer/vulcanize/issues/237 . I wanted to get gulp serve:dist with the starter kit before making any changes then I was going to add the firebase elements which seem to be causing additional issues.

Any help would be appreciated.

Regards,

John
```
**Error**
Johns-MacBook-Pro:polymer-starter-kit-1.1.1 johnwhitton$ gulp
[11:56:13] Using gulpfile ~/Projects/polymer-starter-kit-1.1.1/gulpfile.js
[11:56:13] Starting 'clean'...
[11:56:13] Finished 'clean' after 246 ms
[11:56:13] Starting 'default'...
[11:56:13] Starting 'copy'...
[11:56:13] Starting 'styles'...
[11:56:14] styles all files 98 B
[11:56:14] Finished 'styles' after 534 ms
[11:56:15] copy all files 12.75 MB
[11:56:15] Finished 'copy' after 1.35 s
[11:56:15] Starting 'elements'...
[11:56:15] Finished 'elements' after 3.97 ms
[11:56:15] Starting 'jshint'...
[11:56:15] Starting 'images'...
[11:56:15] Starting 'fonts'...
[11:56:15] Starting 'html'...
[11:56:15] Finished 'fonts' after 81 ms
[11:56:16] Finished 'jshint' after 966 ms
[11:56:16] images all files 35.41 kB
[11:56:16] Finished 'images' after 850 ms
[11:56:16] html all files 52.13 kB
[11:56:16] Finished 'html' after 730 ms
[11:56:16] Starting 'vulcanize'...
[11:56:16] Error in plugin 'gulp-vulcanize'
**Message:
    Error parsing script in /Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/dist/bower_components/iron-selector/iron-selectable.html at NaN:NaN
TypeError: Cannot read property 'push' of undefined**
    at Object.observers (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/ast-utils/declaration-property-handlers.js:59:30)
    at Object.visitors.enterObjectExpression (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/ast-utils/behavior-finder.js:208:33)
    at applyVisitors (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/ast-utils/js-parse.js:28:47)
    at Controller.enter (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/ast-utils/js-parse.js:38:14)
    at Controller.__execute (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/estraverse/estraverse.js:395:31)
    at Controller.traverse (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/estraverse/estraverse.js:493:28)
    at Object.traverse (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/estraverse/estraverse.js:705:27)
    at jsParse (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/ast-utils/js-parse.js:85:14)
    at Analyzer._processScript (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/analyzer.js:384:18)
    at Analyzer.<anonymous> (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/analyzer.js:372:30)
    at Array.forEach (native)
    at Analyzer._processScripts (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/analyzer.js:371:11)
    at Analyzer._parseHTML (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/analyzer.js:301:27)
    at Analyzer.<anonymous> (/Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/node_modules/hydrolysis/lib/analyzer.js:268:22)
    at Timer.listOnTimeout (timers.js:92:15)
Details:
    fileName: /Users/johnwhitton/Projects/polymer-starter-kit-1.1.1/dist/index.html
Johns-MacBook-Pro:polymer-starter-kit-1.1.1 johnwhitton$ 
```

**Comment by johnwhitton** [comment]
Yes fresh clone of the project correct.

**Comment by johnwhitton** [comment]
One thing to note, I downloaded the zip file rather than doing a git clone, let me try a git clone and see what happens

**Comment by johnwhitton** [comment]
This worked ok for me too. However gulp serve:dist did not start a web server (see below)
Peter could you try a gulp serve:dist

*Johns-MacBook-Pro:polymer-starter-kit johnwhitton$ gulp serve:dist
[13:16:12] Using gulpfile ~/Projects/polymer-starter-kit/gulpfile.js
[13:16:12] Starting 'clean'...
[13:16:12] Finished 'clean' after 242 ms
[13:16:12] Starting 'default'...
[13:16:12] Starting 'copy'...
[13:16:12] Starting 'styles'...
[13:16:13] styles all files 98 B
[13:16:13] Finished 'styles' after 514 ms
[13:16:14] copy all files 12.75 MB
[13:16:14] Finished 'copy' after 1.62 s
[13:16:14] Starting 'elements'...
[13:16:14] Finished 'elements' after 5.17 ms
[13:16:14] Starting 'lint'...
[13:16:14] Starting 'images'...
[13:16:15] Starting 'fonts'...
[13:16:15] Starting 'html'...
[13:16:15] Finished 'fonts' after 114 ms
[13:16:16] Finished 'lint' after 1.86 s
[13:16:16] html all files 52.64 kB
[13:16:16] Finished 'html' after 1.05 s
[13:16:16] gulp-imagemin: Minified 6 images (saved 494 B - 1.4%)
[13:16:16] images all files 35.41 kB
[13:16:16] Finished 'images' after 1.21 s
[13:16:16] Starting 'vulcanize'...
Johns-MacBook-Pro:polymer-starter-kit johnwhitton$*

**Comment by garlicnation** [comment]
This was caused by a bug in hydrolysis. Should be fixed with 1.21.3

**Comment by garlicnation** [comment]
If you're still running into this issue run `npm remove hydrolysis && npm install`.
